client/my-sites/email/email-management/home/email-mailbox-action-menu.jsx

```diff
@@ -0,0 +1,260 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+import { recordTracksEvent } from '@automattic/calypso-analytics';
+import { useDispatch, useSelector } from 'react-redux';
+import { useTranslate } from 'i18n-calypso';
+
+/**
+ * Internal dependencies
+ */
+import getCurrentRoute from 'calypso/state/selectors/get-current-route';
+import EllipsisMenu from 'calypso/components/ellipsis-menu';
+import { emailManagementForwarding } from 'calypso/my-sites/email/paths';
+import {
+	getEmailForwardAddress,
+	hasGoogleAccountTOSWarning,
+	isEmailUserAdmin,
+} from 'calypso/lib/emails';
+import {
+	getGmailUrl,
+	getGoogleAdminUrl,
+	getGoogleCalendarUrl,
+	getGoogleDocsUrl,
+	getGoogleDriveUrl,
+	getGoogleSheetsUrl,
+	getGoogleSlidesUrl,
+	hasGSuiteWithUs,
+} from 'calypso/lib/gsuite';
+import {
+	getTitanCalendarlUrl,
+	getTitanContactsUrl,
+	getTitanEmailUrl,
+	hasTitanMailWithUs,
+} from 'calypso/lib/titan';
+import gmailIcon from 'calypso/assets/images/email-providers/google-workspace/services/gmail.svg';
+import googleAdminIcon from 'calypso/assets/images/email-providers/google-workspace/services/admin.svg';
+import googleCalendarIcon from 'calypso/assets/images/email-providers/google-workspace/services/calendar.svg';
+import googleDocsIcon from 'calypso/assets/images/email-providers/google-workspace/services/docs.svg';
+import googleDriveIcon from 'calypso/assets/images/email-providers/google-workspace/services/drive.svg';
+import googleSheetsIcon from 'calypso/assets/images/email-providers/google-workspace/services/sheets.svg';
+import googleSlidesIcon from 'calypso/assets/images/email-providers/google-workspace/services/slides.svg';
+import { hasEmailForwards } from 'calypso/lib/domains/email-forwarding';
+import MaterialIcon from 'calypso/components/material-icon';
+import PopoverMenuItem from 'calypso/components/popover/menu-item';
+import { removeEmailForward } from 'calypso/state/email-forwarding/actions';
+import titanCalendarIcon from 'calypso/assets/images/email-providers/titan/services/calendar.svg';
+import titanContactsIcon from 'calypso/assets/images/email-providers/titan/services/contacts.svg';
+import titanMailIcon from 'calypso/assets/images/email-providers/titan/services/mail.svg';
+
+const removeEmailForwardMailbox = ( { dispatch, mailbox } ) => {
+	recordTracksEvent( 'calypso_domain_management_email_forwarding_delete_click', {
+		destination: getEmailForwardAddress( mailbox ),
+		domain_name: mailbox.domain,
+		mailbox: mailbox.mailbox,
+	} );
+
+	dispatch( removeEmailForward( mailbox.domain, mailbox.mailbox ) );
+};
+
+/**
+ * Returns the available menu items for Titan Emails
+ *
+ * @param {Object} titanMenuParams The argument for this function.
+ * @param {string} titanMenuParams.email The email address of the Titan account
```

Comment:
I didn't know we could specify parameters like this 😯.

Reply:
I referenced jsdoc.com to work out how to do this!